Rakefile

```diff
@@ -5,4 +5,4 @@ require File.expand_path("config/application", __dir__)
 
 Rails.application.load_tasks
 
-task default: %i[spec]
+task default: %i[lint spec]
```

Comment:
We should consider using the approach [here](https://github.com/alphagov/content-publisher/blob/master/Rakefile#L7), as I think this will still run `spec` first.

Reply:
I did not realise that! Thanks.

```suggestion
# RSpec shoves itself into the default task without asking, which confuses the ordering.
# https://github.com/rspec/rspec-rails/blob/eb3377bca425f0d74b9f510dbb53b2a161080016/lib/rspec/rails/tasks/rspec.rake#L6
Rake::Task[:default].clear
task default: %i[lint spec]
```